dask/array/routines.py

```diff
@@ -253,7 +253,7 @@ def vdot(a, b):
     return dot(a.conj().ravel(), b.ravel())
 
 
-@wraps(np.matmul)
+@safe_wraps(getattr(np, 'matmul'))
```

Comment:
Why `getattr` here rather than `np.matmul`?

Reply:
i'm really not sure. I thought you had to for this function.

Reply:
squash merge please `np.matmul` works.

Reply:
Will do.  (we squash by default)

Reply:
Unrelated question, How come I have the power to create new repos for dask, but not merge PRs here? Was that intentional by you?

Reply:
Currently I believe that you have membership rights in the organization, but not commit privileges in any particular repository.

Reply:
Got it! I didn't know it could be that granular.

Reply:
Currently the best source of information on membership policies is here: https://github.com/dask/dask/issues/3223#issuecomment-371475223